src/templates/_includes/app_preview.html

```diff
@@ -11,10 +11,10 @@
   <h1 class="name">{{ app.name }}</h1>
   <div class="provider">
     {% if app.author %}
-      <span>by</span>
+      <span>{{ _('by') }}</span>
```

Comment:
This should be one string "by Author Name" and not split into "by" and then Author such since in some languages like Japanese the attribution "de" is appended to the end of the author's name.